packages/core/integration-tests/test/integration/babelrc-custom/babel-plugin-dummy.js

```diff
@@ -0,0 +1,13 @@
+module.exports = () => {
```

Comment:
This is derived from https://github.com/parcel-bundler/parcel/blob/d12f38120872171c6f6dfe5586d838287b49a5d8/packages/core/integration-tests/test/integration/babel-config-js/node_modules/babel-plugin-dummy/index.js